code/hello/README.md

```diff
@@ -3,15 +3,23 @@
 
 This example should help to check that your machine is well installed.
 
-## g++ & make
+## make vs cmake
 
-Try:
+On any linux like system, provided you have a "recent enough" g++, this should work out of the box:
 ```
 make
-export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:.
 ./hello
 ```
 
+On  native Windows, it may be mandatory to run `cmake` first:
+```
+mkdir build
+cd build
+cmake ..
+cmake --build .
+Debug/hello.exe
+```
```

Comment:
Since you already describe how to build on Windows, you could also add:

Alternatively, after `cmake ..`, you can open the `hellow.sln` in Visual Studio and press F5, which builds and launches the program.

Reply:
I want to add something like that now with #124.